pkgs/tools/security/passage/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, rustPlatform, installShellFiles, asciidoctor, xorg, python3, passAlias ? false }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "passage";
+  version = "0.1.0";
+
+  src = builtins.fetchGit {
+    url = "https://git.sr.ht/~gpanders/passage";
+    ref = "master";
+    rev = "9bddfe8e03b39ceccfa736e88924d3b4cb59748f";
+  };
+
+  cargoSha256 = "06clfavpxc7a11jxml0vkw8x7lya5kipv8xyi59f8fqsksj79k8m";
+
+  nativeBuildInputs = [ asciidoctor python3 installShellFiles ];
+  buildInputs = [ xorg.libxcb.dev ];
```

Comment:
Did you try if xorg.libxcb is enough? dev should be added automatically.

Reply:
Yeah, you're right xorg.libxcb is indeed enough. Hadn't checked.